packages/retail-ui-extensions/src/extension-api/CartApi/CartApiContent.ts

```diff
@@ -0,0 +1,102 @@
+import type {Address, Cart, Customer, CustomSale} from '../Models/Cart';
```

Comment:
Just as a nitpicky note: I do not think we should use pascal case names unless we are talking about components. In the end it doesn't matter to consumers, but my original intention was for pascal case to signal a similar function as it does in our React (web) codebases, where it almost always means "component" (with everything else using kebab case, or sometimes camel case)